### DISCUSSION TITLE
GitHub Feedback built on "Discussions" feels strange and wrong

### DISCUSSION THREAD

**Discussion by CreepyGnome** [community/community#5311] [Other Feature Feedback, Questions, & Ideas]
This really feels like a square peg in a round whole situation. As "feedback" isn't typically rating answers and marking somethign as "answered".  Its about trying to make a company in this case aware of complaints, issues, or even giving at-a-boys, and hoping the company will provide a response maybgin stating they are fixing and allowing others to vent or add to it, sure is a discussion, but its not "answers".

So answers in feedback here tend to be more like "threads" of thought for people who want to add to the feedback in some way.  Why would a thread on feedback be marked as answered? 

Seems like you could easily repurpose discussions to really be a true feedback feature in its own right to allow for people to vote on feedback the feel is important and allow for thread conversations on the feedback, but that the feedback is never "answered" but maybe it could be "resolved" by GitHub when they make a formal bug or feature request in the appropriate areas, and link it.  

As it is this feedback system within the discussions feature feels really strange and wrong.

**Comment by dserodio** [comment]
I have the feeling that GitHub staff doesn't read these "Discussions" 😞
